hibernate-core/src/main/java/org/hibernate/type/descriptor/jdbc/NClobJdbcType.java

```diff
@@ -46,19 +46,34 @@ public <X> ValueExtractor<X> getExtractor(final JavaType<X> javaType) {
 		return new BasicExtractor<>( javaType, this ) {
 			@Override
 			protected X doExtract(ResultSet rs, int paramIndex, WrapperOptions options) throws SQLException {
-				return javaType.wrap( rs.getNClob( paramIndex ), options );
+				if (options.getDialect().supportsNationalizedMethods()) {
```

Comment:
```suggestion
				if ( options.getDialect().supportsNationalizedMethods() ) {
```

The convention in the Hibernate codebase is to avoid stacking parentheses together, and to add whitespace like this.

Reply:
Ok, I will change it